native/cocos/core/scene-graph/Node.h

```diff
@@ -471,8 +471,7 @@ class Node : public CCObject {
         _hasChangedFlags = value;
     }
 
-    inline void setDirtyFlag(uint32_t value) { _dirtyFlag = value; }
-    inline uint32_t getDirtyFlag() const { return _dirtyFlag; }
+    inline bool isTransformDirty() const { return _transformFlags != static_cast<uint32_t>(TransformBit::TRS); }
```

Comment:
I think the logic is not correct. For example, `TransformBit::POSITION != TransformBit::TRS`, but it is dirty. Can use `_transformFlags  ^| TransformBit::None`, or `_transformFlags  != TransformBit::None`.

Reply:
fixed